tests/test_generate_project.py

```diff
@@ -396,3 +399,29 @@ def test_django_package_no(
             "run: tox -f py$(echo ${{ matrix.python-version }} | tr -d .)",
         ],
     )
+
+
+def test_add_me_as_contributor(
+    tmp_path: Path,
+    base_answers: dict[str, str | bool],
+):
+    if CI and platform == "darwin":
+        pytest.skip("Skipping due to APi rate limit")
```

Comment:
Turns out `all-contributors-cli add` makes some API calls to GitHub to resolve some details based on the username. Could we remove this test from CI or run it only once?